tests/test_unit_conversion.py

```diff
@@ -711,26 +710,32 @@ def test_convert_variable_invalid_arguments(self, local_model):
         # arguments wrong types
         with pytest.raises(AssertionError):
             local_model.convert_variable('x', unit, direction)
+        local_model.convert_variable('x', unit, direction, raise_errors=False) == 'x'
```

Comment:
There are asserts missing here. But in any case I don't think we want this feature.